packages/ember-metal/lib/observer_set.js

```diff
@@ -1,66 +1,37 @@
-import { guidFor } from 'ember-utils';
-import { sendEvent } from './events';
 
-/*
```

Comment:
This comment was useful (in general) for explaining the internal data structure used in `ObserverSet`.  Can you bring back a new comment that shows what things look like now?